### PR TITLE
update integration

### PR DESCRIPTION
**Pull request by AleksanderWWW**

### CODE REVIEW

File: src/neptune_lightgbm/impl/__init__.py [hunk 1]
```diff
@@ -31,24 +31,32 @@
...
 
+from neptune_lightgbm.impl.version import __version__
+
 try:
-    # neptune-client=0.9.0+ package structure
-    import neptune.new as neptune
-    from neptune.new.integrations.utils import (
+    from neptune import Run
+    from neptune.handler import Handler
+    from neptune.ntegrations.utils import (
```
Comment (kshitij12345) [inline src/neptune_lightgbm/impl/__init__.py]: typo -> integrations

File: tests/test_e2e.py [hunk 2]
```diff
@@ -3,20 +3,19 @@
 import pytest
 
 try:
-    # neptune-client=0.9.0+ package structure
-    from neptune.new import init_run
```
Comment (kshitij12345) [inline tests/test_e2e.py:7]: For test, do we need try catch as we only test with version >1.0?

Comment (AleksanderWWW) [reply tests/test_e2e.py:7]: As I discussed with @Raalsky earlier, it might be possible that a user wants to run the tests locally

**Review by kshitij12345 — APPROVED** [review]
LGTM! Thanks @AleksanderWWW

**Pull request merged**